lib/utils.js

```diff
@@ -56,6 +56,22 @@ module.exports.getAuthHeader = function getAuthHeader(timestamp, apiKey, apiSecr
   return header.join(', ');
 };
 
+module.exports.prepareInitialRequest = function prepareInitialRequest() {
+  var request = {};
+  var nonEnumberables = ['ip'];
```

Comment:
This could be passed as an argument, but I wasn't sure if that's worth it, as it's one-off util.